src/applications/personalization/dashboard/components/benefit-payments-v2/BenefitPaymentsV2.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { subDays } from 'date-fns';
+import PaymentsCardV2 from './PaymentsCardV2';
+import DashboardWidgetWrapper from '../DashboardWidgetWrapper';
+import IconCTALink from '../IconCTALink';
+import recordEvent from '~/platform/monitoring/record-event';
+
+const NoRecentPaymentText = () => {
+  return (
+    <p
+      className="vads-u-margin-bottom--3 vads-u-margin-top--0"
+      data-testid="no-recent-payments-paragraph-v2"
+    >
+      You haven’t received any payments in the past 30 days.
+    </p>
+  );
+};
+
+const PopularActionsForPayments = ({ showPaymentHistoryLink = false }) => {
+  return (
+    <>
+      <h3 className="sr-only">Popular actions for Benefit Payments</h3>
+      {/* todo check for direct deposit first */}
+      <IconCTALink
+        href="/profile/direct-deposit"
+        icon="dollar-sign"
+        text="Manage your direct deposit"
+        /* eslint-disable react/jsx-no-bind */
+        onClick={() => {
+          recordEvent({
+            event: 'nav-linkslist',
+            'links-list-header': 'Manage your direct deposit',
+            'links-list-section-header': 'Direct deposit',
+          });
+        }}
+        /* eslint-enable react/jsx-no-bind */
+        testId="manage-direct-deposit-link-v2"
+      />
+      {showPaymentHistoryLink && (
+        <IconCTALink
+          href="/va-payment-history/payments/"
+          icon="user-check"
+          text="View your payment history"
+          /* eslint-disable react/jsx-no-bind */

```

Comment:
I'm torn on this, since I understand why its disabled, but also if it is worth just moving these recordEvents into an object up above specifically for analytics handlers to satisfy eslint and also to stay with the code style that is used elsewhere for click handlers.

Maybe something like this https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/appeals/10182/components/AddIssue.jsx#L103